Portfolio1.html

```diff
@@ -0,0 +1,129 @@
+	
+			<h3>Sub-heading 2</h3>
+				<h4>19th of January 2021</h4>
+					<section> Doctor and Student explores the relationship between the common law and equity and distinguishes a number of sources of legal principles. It was an important text for English law students at least until William Blackstone's Commentaries on the Laws of England was published in the mid-18th century. Alexander II's rise is connected to the dynastic feuds of the Seleucid Empire. Both King Seleucus IV (d. 175 BC) and his brother Antiochus IV (d. 164 BC) had descendants contending for the throne, leading the country to experience many civil wars. The situation was complicated by Ptolemaic Egyptian interference, which was facilitated by the dynastic marriages between the two royal houses. In 128 BC, King Demetrius II of Syria, the representative of Seleucus IV's line, invaded Egypt to help his mother-in-law Cleopatra II who was engaged in a civil war against her brother and husband King Ptolemy VIII. Angered by the Syrian invasion, the Egyptian king instigated revolts in the cities of Syria against Demetrius II and chose Alexander II, a supposed representative of Antiochus IV's line, as an anti-king. With Egyptian troops, Alexander II captured the Syrian capital Antioch in 128 BC and warred against Demetrius II, defeating him decisively in 125 BC. The beaten king escaped to his wife Cleopatra Thea in the city of Ptolemais, but she expelled him. He was killed while trying to find refuge in the city of Tyre.</section>
+	</article>
+
+<!--Images and their Titles -->
+	
+	<!--Sub-Image 1 -->
+	<div class="images">
+		<div class="sub-image-title-formatting1">
+			<h3 class="sub-image-title1">Kings</h3>
+				<a target="_blank" href="https://en.wikipedia.org/wiki/Arminius">
+					<img src="pexels-csongor-kemeny-3933541.jpg" alt="Image of an iron statue of a regal looking king with a halo behind his head situated amongst the trees in cloud, click here for information on Arminius" class="sub-images">
+					</a>
+		</div>
+		
+	<!--Sub-Image 2 -->	
+		<div class="sub-image-title-formatting2">
+			<h3 class="sub-image-title2">Flyers</h3>
+				<a target="_blank" href="https://www.oddee.com/item_98487.aspx">
+					<img src="pexels-pixabay-61155.jpg" alt="Image of a brightly coloured bird head with a red head, purple, blue and orange plumage and a large blue eye, click here for more information on exotic birds" class="sub-images">
+				</a>
+		</div>
+		
+	<!--Sub-Image 3 -->
+		<div class="sub-image-title-formatting3">
+			<h3 class="sub-image-title3">Jokers</h3>
+				<a target="_blank" href="https://www.thesprucecrafts.com/best-card-trick-2267052">
+					<img src="pexels-oleg-magni-1831111.jpg" alt="Image of a hand of six cards spread out on a yellow background with a vibrant joker in the fore, click here for more information on card tricks" class="sub-images">
+				</a>
+		</div>
+	</div>
+</div>
+
+<!--Second Button Bar -->
+<div class="buttons1">
```

Comment:
might be better to semantically call this navButtons

Reply:
Done!